codespell_lib/data/dictionary.txt

```diff
@@ -24601,8 +24712,13 @@ reposititioning->repositioning
 repositorry->repository
 repositry->repository
 reposoitory->repository
+repost->riposte
+reposted->riposted
+repostes->ripostes
+reposting->riposting
 repostiories->repositories
 repostiory->repository
+reposts->ripostes
```

Comment:
reposts is correct, so this should be removed: https://www.merriam-webster.com/dictionary/repost